universal-application-tool-0.0.1/app/controllers/LegacyRoutesController.java

```diff
@@ -11,14 +13,14 @@
 public class LegacyRoutesController {
 
   public Result idcsLoginWithRedirect(Http.Request request, Optional<String> redirectTo) {
```

Comment:
Should the request be used in the body? I see it's dropped here.  Is the intent to always redirect to the new handler rather than handle the old route through new refactored flow?

Reply:
If I understand correctly, I think the `redirect` method used automatically passes in the request context. Here's the docs on it: https://www.playframework.com/documentation/2.8.x/JavaRouting#Reverse-routing

Reply:
Yes Shubha that was my understanding as well! Thanks for the docs! I can add them as a comment if you think it would be helpful Shane?

Reply:
Great! thanks yall!